.env.tmp

```diff
@@ -18,6 +18,13 @@ CONTAINER_REGISTRY_SERVER="localhost:5000"
 CONTAINER_REGISTRY_USERNAME=""
 CONTAINER_REGISTRY_PASSWORD=""
 
+    # To specify additional container registries ensure the prefix is CONTAINER_REGISTRY_SERVER, CONTAINER_REGISTRY_USERNAME, CONTAINER_REGISTRY_PASSWORD
+    # And the token following the prefix uniquely associates the SERVER/USERNAME/PASSWORD
```

Comment:
Very minor, but can we explicitly call out for the reader whether the token must be numeric, versus a string like "CORPORATE"?